annis-visualizers/src/main/java/annis/visualizers/htmlvis/HTMLVis.java

```diff
@@ -108,7 +119,11 @@ public Panel createComponent(VisualizerInput vi, VisualizationToggle vt)
 
   
     String visConfigName = vi.getMappings().getProperty("config");
-  
+    String hitMarkConfig = vi.getMappings().getProperty("hitmark","on");
```

Comment:
The "hitmark" mapping is not documented in the javadoc of this class (https://github.com/korpling/ANNIS/pull/451/files#diff-ad784e15aef356625aa32b2bcf9e9dc6R66).